kuma/wiki/tests/test_views.py

```diff
@@ -339,17 +339,23 @@ def test_footer_contributors(self, get_contributors, flag_is_active):
         # just checking if the contributor link is rendered
         eq_(len(contributors.find('a')), 2)
 
-    def test_revision_view_bleached_content(self):
-        """Bug 821988: Revision content should be cleaned with bleach"""
-        d, r = doc_rev("""
-            <a href="#" onload=alert(3)>Hahaha</a>
-            <svg><svg onload=alert(3);>
-        """)
+    def test_template_revision_content(self):
+        doc = document(title='Testing Template', slug='Template:Testing', save=True)
+        r = revision(save=True, document=doc, is_approved=True)
+
         resp = self.client.get(r.get_absolute_url())
-        page = pq(resp.content)
-        ct = page.find('#wikiArticle').html()
-        ok_('<svg>' not in ct)
-        ok_('<a href="#">Hahaha</a>' in ct)
+
+        ok_('Revision Source' in resp.content)
+        ok_('Revision Content' not in resp.content)
+
+    def test_article_revision_content(self):
+        doc = document(title='Testing Article', slug='Article', save=True)
+        r = revision(save=True, document=doc, is_approved=True)
+
+        resp = self.client.get(r.get_absolute_url())
+
+        ok_('Revision Source' in resp.content)
+        ok_('Revision Content' in resp.content)
```

Comment:
I think it might be nice to include a test for whether the content has `open="open"` as well. Generally I look for things that are conditional for testing.